File: SiteParserWEB/src/app/core/models/result.model.ts
export class ResultModel {
    id: number;
    sentence: string;
    word: string;
    count: number;
    constructor(data?){
        if (data == null || data === 'undefined') {
            return;
        }
        this.id = data.id;
        this.sentence = data.sentence || '';
        this.word = data.word || '';
        this.count = data.count || '';
    }
}

File: SiteParser.API/SiteParser.API/Models/ResultModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Threading.Tasks;

namespace SiteParser.API.Models
{
    public class ResultModel
    {
        [Key]
        public int ID { get; set; }
        public string Sentence { get; set; }
        public string Word { get; set; }
        public int Count { get; set; }
    }
}
